### DISCUSSION TITLE
github actions issue after switch to freeplan

### DISCUSSION THREAD

**Discussion by caner-aras** [community/community#131770] [Actions]
### Select Topic Area

Question

### Body

hi, i deleted my registered credit card details and i switched to free plan. (i had copilot)  since switch on freeplan, my actions not works. Can i use actions on my repos? i mean, i created workflows on actions on some repos. and actions are not working since the billing issue. By the way, I'm not at my limit, it's only 2000/25 mins.

i see the error message "[Error](https://github.com/xxxx/workflow)
The job could not be started because the last account payments failed or your spending limit needs to be increased. Please check the 'Billing and plans' section in your settings."

does anyone have any suggestions

**Comment by paiineli** [comment]
Hi,

It sounds like you're encountering issues with GitHub Actions after switching to the free plan. Since you deleted your credit card details, it might have affected your ability to use certain features.

Here are some suggestions to resolve the issue:

Check Billing Settings: Go to your "Billing and plans" section in GitHub settings. Ensure that your account is set up correctly and that there are no outstanding issues.

Spending Limit: If your spending limit is set too low, consider increasing it. Even on a free plan, some features might require adjustments to your limits.

Contact Support: If everything appears correct but actions are still not working, reach out to GitHub Support for assistance. They can help clarify any issues with your account or billing status.

Review Workflow Configurations: Double-check your workflow configurations to ensure there are no errors that might prevent jobs from starting.

I hope this helps, and you can get your GitHub Actions working again soon!

Best,
